docs(blog): add contributor highlight Philippe Scorsolini (#288)
cloudnative-pg/cloudnative-pg.github.io: content/blog/contributor-highlight-philippe-scorsolini/index.md

---
title: "Contributor Spotlight: Philippe Scorsolini"
date: 2025-05-05T14:51:43+02:00
draft: false
image:
    url: philippe.png
    attribution:
author: fdrees
tags:
 - prometheus
 - pooler
 - kubernetes
 - postgresql
 - open-source
summary: "In a mini-series on this blog we highlight the work of the community.
  Today we meet Philippe Scorsolini, Staff Software Engineer and Tech Lead at Upbound,
  and CloudNativePG maintainer."
---

Building and maintaining and open source project takes a village. In a
mini-series on this blog we would like to highlight the work of our
maintainers, component owners, and members of the larger community.

Today's spotlight: [Philippe Scorsolini](https://github.com/phisco/), 
based in Italy. Philippe is a Staff Software Engineer and Tech Lead at 
Upbound, working on their Spaces product, and maintaining the Crossplane 
project.

Coming from an infrastructure background, Philippe focused on Kubernetes 

> and only used PostgreSQL as a user, not as an administrator. "However, 

Reviewer: user - user. I'd vary ti
```suggestion
and knew PostgreSQL as a user, but not as an administrator. "However, 
```

I was always fascinated by PostgreSQL!" So, when the team at EDB reached 
out for a position working on a Kubernetes controller for it, he accepted 
right away. That operator of course is now known as CloudNativePG. "Once 

> it was open-sourced, although I wasn’t working on it full-time anymore, 
> they honored me by making me a maintainer."

Reviewer: I think "honor me" sounds like they sacrificed animals for Philippe.
```suggestion
it was open-sourced, although I wasn’t working on it full-time anymore, 
they gave me the great honor of making me a maintainer."
```

His first contribution to the project was small, just adding a name to 
the metrics port to allow it to be easily discovered by the Prometheus 

> operator. The next one was more interesting since Philippe was one of the 

Reviewer: ```suggestion
operator. The next one was more interesting: Philippe was one of the 
```
